tests/terratest/functions/set/setConfigTF.go

```diff
@@ -0,0 +1,52 @@
+package functions
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/v2/validation/terratest"
+)
+
+func SetConfigTF(k8sVersion string, nodePools []terratest.Nodepool) (done bool, err error) {
+	terraformConfig := new(terratest.TerraformConfig)
+	config.LoadConfig("terraform", terraformConfig)
+
+	module := terraformConfig.Module
+
+	var result bool
+	var file *os.File
+
+	file, err = os.Create("../../../../terratest/modules/cluster/main.tf")
```

Comment:
I just feel uncomfortable with a relative path here. If it will be a constant, we should define it as such imo and reference the const here

Reply:
Why do you feel uncomfortable w/ a relative path here??  please help me understand the concern :)

Reply:
I will implement your suggestion, but I'd like to understand the reasoning behind the concern.

Reply:
it may not be a big deal, but it isn't obvious to me as someone maintaining the code where that file is. But regardless, minimum change would be to make it a const.

Reply:
Sure thing, i'll make the change.  Thanks Caleb